Birdboat: atmos split and small tweaks

coiax opened the pull request:
- Birdboat has a drone shell dispenser
- Supply loop is separated into a north and south loop, which
  are not connected, south loop has its own supply tanks. Both loops
  are also separated by new airlocks.
- Expanded shuttle dock numbers (no change physically, but updated
  numbers to inform what can dock successfully)
- Medical has an experimental space cooling system for cryo
- Moved disused medical freezer into maint
- Added mime can to Theatre, metal foam grenades to maint
- Wooden barricades have been added between the spider pen room and the nearest vent

:cl: coiax
rscadd: Birdboat's supply loop has been split into two unconnected halves, to maintain award winning air quality.
/:cl:

coiax committed:
Improvments to Birdboat atmos plus others

- Birdboat has a drone shell dispenser
- Supply loop is seperated into a north and south loop, which
are not connected, south loop has its own supply tanks. Both loops
are also seperated by new airlocks.
- Expanded shuttle dock numbers (no change physically, but updated
numbers to inform what can dock successfully)
- Medical has an experimental space cooling system for cryo
- Moved disused medical freezer into maint
- Added mime can to Theatre, metal foam grenades to maint

Zilenan91 commented:
I think birdboat already had a drone dispenser in maint to the right of the spider temple

coiax committed:
Spider safety

Put a wooden baricade between the spider pen and the nearest vent. Surely a spider couldn't break this solid wooden baricade!

coiax commented:
It doesn't.

PKPenguin321 commented:
Are you sure the atmos tanks actually work? Last I checked you couldn't actually get gasses out of them
